### ISSUE TITLE
OBS does not take into account multiple network cards

### ISSUE THREAD

**Issue by xDreams0** [obsproject/obs-studio#6199]
### Operating System Info

Windows 11

### Other OS

No

### OBS Studio Version

27.2.3

### OBS Studio Log URL

https://obsproject.com/logs/uzlk2oQvzxISGR__

### Expected Behavior

Hello, wishing to use another network to broadcast, I decided to take another network card and linked it via OBS to the broadcasts. This one does not work correctly despite the procedures done via the help on Discord.

### Current Behavior

OBS doesn't want to send the stream to the second, it starts with the first and gets stuck.

### Steps to Reproduce

1. Connect second network card
2. OBS Settings
3. Advanced settings
4. Select other card
5. Start to stream
6. The connection timed out. Try a different server, or check that the connection is not being blocked by a firewall or other security software (ETIMEDOUT).

**Comment by kkartaltepe** [comment]
It looks like the user solved the issue by successfully installing their network drivers.

**Comment by xDreams0** [comment]
Good evening,
I don't think the problem is solved.
I just redid the tests, the metric was not concluded. It's my secondary network that took everything, the first got nothing on the line.
I performed a connection sharing test with my smartphone by assigning it a metric and it no longer works. I can't say if it's related to OBS or Windows 11, but what is sure is that there is a metric problem somewhere.

Having watched a lot of videos on implementing what I'm looking to do, they all use the metric and it works fine. Could this be a problem with OBS not working with Windows 11?

**Comment by mufunyo** [comment]
You can use the Bind to IP function in the advanced settings to tell OBS to use a specific network adapter.

**Comment by xDreams0** [comment]
Good evening,
@mufunyo 
Thank you for your comment, I had already done the manipulation, hence the opening of this ticket.

I just did a test on a computer with Windows 10. I did not encounter any problems with OBS and 2 network cards. Do it with the metric fine tune.

So I decided to do a final test on my computer with Windows 11, and I reset the network cards in general, they are like new, both the main and the secondary. And miraculously it works like new.
I think OBS should indicate that if there is a problem with the aggregation of a secondary card, to try resetting the cards so that Windows manages the metrics correctly and that OBS does not get lost in the communication of this information from Windows.


In any case, thank you all for your advice, I will be able to stream correctly via my second line without impacting the main one ;)
